### PR TITLE
sponsor: add sponsor view

### PR DESCRIPTION
**Pull request by pguenezan**
- still need to rework the page style

### CODE REVIEW

File: prologin/prologin/templates/prologin/main-menu.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
   <li class="{% active '^/train/' %}"><a href="{% url 'problems:index' %}">{% trans "Train" %}</a></li>
   <li class="{% active '^/archives/' %}"><a href="{% url 'archives:index' %}">{% trans "Archives" %}</a></li>
   <li class="{% active '^/forum/' %}"><a href="{% url 'forum:index' %}">{% trans "Forums" %}</a></li>
+  <li class="{% active '^/sponsors/' %}"><a href="{% url 'sponsors:index' %}">{% trans "Sponsors" %}</a></li>
```
Comment (seirl) [inline prologin/prologin/templates/prologin/main-menu.html]: I know this isn't final, but I really don't want to add too much clutter to that menu. I'd rather have a "See all the sponsors" button on the homepage.

**Review by seirl — CHANGES_REQUESTED** [review]

File: prologin/documents/templates/documents/interviews.tex [hunk 2]
```diff
@@ -77,7 +77,6 @@
         \item Projets perso ?
         \item Nouveaux projets depuis l'année dernière ?
         \item Nous parler de quelque chose en info ?
-        \item Hobbies non informatiques ?
```
Comment (seirl) [inline prologin/documents/templates/documents/interviews.tex]: What is this doing here? And why?

File: prologin/homepage/templates/homepage/sponsors.html [hunk 3]
```diff
@@ -14,3 +15,11 @@ <h2>{% trans "Sponsors" %}</h2>
   </div>
   {% endfor %}
 </div>
+
+<br/>
+
+<p>
+  <a href="{% url 'sponsors:index' %}" class="btn btn-default btn-block">
```
Comment (seirl) [inline prologin/homepage/templates/homepage/sponsors.html]: Don't use a button for that, it's a link, it should look like a link.

File: prologin/prologin/urls.py [hunk 4]
```diff
@@ -67,6 +67,9 @@ def crash_test(request):
     # Schools
     path('schools/', include('schools.urls', namespace='schools')),
 
+    # Sponsors
+    path('sponsors/', include('sponsor.urls', namespace='sponsors')),
```
Comment (seirl) [inline prologin/prologin/urls.py:71]: Why would you create a whole new module for a single static page?

File: prologin/sponsor/templates/sponsor/index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    {{ block.super }}
+    {% breadcrumb_for "sponsors:index" %}{% trans "Sponsors" %}{% endbreadcrumb_for %}
+{% endblock breadcrumbs %}
+
+{% block title %}{% trans "Sponsors" %}{% endblock %}
+
+{% block content %}
+
+    <h1>{% trans "Sponsors" %}</h1>
+    <p>{% blocktrans %}Looking for Prologin's last year poster? 1996 final's statement? 2003 quiz?
```
Comment (seirl) [inline prologin/sponsor/templates/sponsor/index.html]: wat

File: prologin/sponsor/templates/sponsor/index.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    You can find all the documents and photo albums from past Prologin editions in these pages.{% endblocktrans %}</p>
+
+    {% for sponsor in sponsors %}
+        <hr/>
+        <div class="row">
+            <div class="col-sm-6 col-md-3">
+                <a href="{{ sponsor.site }}" style="width: 100%; display: block; text-align: center">
```
Comment (seirl) [inline prologin/sponsor/templates/sponsor/index.html]: Don't hardcode style in the HTML template

**Review by seirl — CHANGES_REQUESTED** [review]

File: prologin/homepage/templates/homepage/sponsors.html (hunk 3 above)
Comment (remi-dupre) [reply prologin/homepage/templates/homepage/sponsors.html]: I tried several design, the only feedback I got was for the opposite :/

File: prologin/prologin/urls.py (hunk 4 above)
Comment (remi-dupre) [reply prologin/prologin/urls.py:71]: There was already a `sponsors` module holding related models, it seems it should be the right place to incorporate this page?

Comment (seirl) [reply prologin/prologin/urls.py:71]: Ah, okay then 👍

**Pull request merged**